@Types/ITechRecord.d.ts

```diff
@@ -82,54 +85,60 @@ interface Axle {
     };
 }
 
-export interface AdrDetails {
-    vehicleDetails: VehicleDetails;
-    permittedDangerousGoods: PERMITTED_DANGEROUS_GOODS[];
-    additionalExaminerNotes: string;
-    applicantDetails: ApplicantDetails;
-    memosApply: MEMOS_APPLY[];
-    additionalNotes: AdditionalNotes;
-    tank: Tank;
-}
-
-export interface AdditionalNotes {
-    number: number;
-    guidanceNotes: string[];
-}
-
-export interface ApplicantDetails {
-    name: string;
-    street: string;
-    town: string;
-    city: string;
-    postcode: string;
-}
-
-export interface Tank {
-    tankDetails: TankDetails;
-    tankStatement: TankStatement;
-}
-
-export interface TankDetails {
-    tankManufacturer: string;
-    tc2IntermediateApprovalNo: string;
-    tc2IntermediateExpiryDate: Date;
-    tc3PeriodicNumber: string;
-    tc3PeriodicExpiryDate: Date;
-    yearOfManufacture: string;
-    tankCode: string;
-    specialProvisions: string;
-    tankManufacturerSerialNo: string;
-    tankTypeAppNo: string;
-}
-
-export interface TankStatement {
-    substancesPermitted: SUBSTANCES_PERMITTED;
-    statement: string | null; // statement and productList are mutually exclusive
-    productList: string | null;
+interface AdrDetails {
+    vehicleDetails: {
+        type: string,
+        approvalDate: string
+    },
+    listStatementApplicable?: boolean,
+    batteryListNumber?: string,
+    declarationsSeen?: boolean,
+    brakeDeclarationsSeen?: boolean,
+    brakeDeclarationIssuer?: string,
+    brakeEndurance?: boolean,
+    weight?: string,
+    compatibilityGroupJ?: boolean,
+    documents?: string[],
+    permittedDangerousGoods: string[],
+    additionalExaminerNotes?: string,
+    applicantDetails: {
+        name: string,
+        street: string,
+        town: string,
+        city: string,
+        postcode: string
+    },
+    memosApply?: string[],
+    additionalNotes?:{
+        number?: string[],
+        guidanceNotes?: string[]
+    },
+    adrTypeApprovalNo?: string,
+    tank?:{
+        tankDetails?:{
```

Comment:
this tankDetails object being restructured will also break the mobile app

Reply:
as discussed this is a minimal change , although understand this will impact VTA mobile app. @vladddragomir  will update the stakeholders.